src/bin/mdbook-epub.rs

```diff
@@ -39,7 +43,10 @@ fn run(args: &Args) -> Result<(), Error> {
     // get a `RenderContext`, either from stdin (because we're used as a plugin)
     // or by instrumenting MDBook directly (in standalone mode).
     let ctx: RenderContext = if args.standalone {
-        let md = MDBook::load(&args.root).map_err(SyncFailure::new)?;
+        let book_root_clone = args.root.clone();
+        let error = format!("book.toml root file is not found by a path {:?}",
+                        &book_root_clone.into_os_string().into_string().to_owned().unwrap());
+        let md = MDBook::load(&args.root).expect(&error);
```

Comment:
Why use `expect()` in a function that returns a `Result`?

Reply:
I suppose that is the new mdbook 0.4.4 version changes.
Now **MDBook::load(&args.root)**  returns Result<T,E> and the next line : **md.build_dir_for("epub");**  expects unwrapped instance in code. So ok, let's unwrap() result from .load()

```
let md = MDBook::load(&args.root).unwrap();
let destination = md.build_dir_for("epub");
```
That's very easy to run utility with incorrect path and get unclear error. So I decided to make displaying error more clear. We want to have a path to 'book.toml' in reality, but we get confusing following error, like below:

I have a local book sources in :
/media/ylarin/PHOTO/rust_projects/rust-book-ru-blangder/rustbook-ru  (book.toml is here) 
/media/ylarin/PHOTO/rust_projects/rust-book-ru-blangder/rustbook-ru/src  (SUMMARY.md is here)

1. Let's make an incorrect run like that. I replaced .expect(x) by unwrap() here.
```
**./mdbook-epub -s /media/ylarin/PHOTO/rust_projects/rust-book-ru-blangder**
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Couldn't open SUMMARY.md
Caused by:
    No such file or directory (os error 2)', src/bin/mdbook-epub.rs:50:43
```
So why does it looking for SUMMARY.md ??  

2. Ok, let's supply that folder with summary,md instead:
**$ ./mdbook-epub -s /media/ylarin/PHOTO/rust_projects/rust-book-ru-blangder/rustbook-ru/src**
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Couldn't open SUMMARY.md
Caused by:
    No such file or directory (os error 2)', src/bin/mdbook-epub.rs:50:43

So again the error message is very confusing.

3. The code:
`let md = MDBook::load(&args.root).expect(&error);`
solves that confusing message like that:
**./mdbook-epub -s /media/ylarin/PHOTO/rust_projects/rust-book-ru-blangder/rustbook-ru/src**
thread 'main' panicked at 'book.toml root file is not found by a path "/media/ylarin/PHOTO/rust_projects/rust-book-ru-blangder/rustbook-ru/src": Couldn't open SUMMARY.md
Caused by:
    No such file or directory (os error 2)', src/bin/mdbook-epub.rs:49:43


_In general I would like to add more logging inside code, so it would be possible to gather people's log to identify possible problems more easier._